website/docs/d/pi_volume_onboardings.html.markdown

```diff
@@ -40,10 +40,11 @@ Review the argument references that you can specify for your data source.
 ## Attribute reference
 In addition to all argument reference list, you can access the following attribute references after your data source is created. 
 
-- `onboardings` - (List of objects) - The list of volume onboardings.
+- `onboardings` - (List) List of volume onboardings.
 
   Nested scheme for `onboardings`:
       - `description` - (String) The description of the volume onboarding operation.
       - `id` - (String) The type of cycling mode used.
-      - `input_volumes` - (List of strings) List of volumes requested to be onboarded.
-      - `status` (String) Tthe status of volume onboarding operation.
\ No newline at end of file
+      - `input_volumes` - (List) List of volumes requested to be onboarded.
+      - `status` (String) The status of volume onboarding operation.
+      
```

Comment:
nit: Delete this extra line.

Reply:
It's actually quite uneven currently whether markdown files end with a newline or not. Would you prefer they don't?

Reply:
If they don't end with newline it shows as a warning sign. Here it is showing a line-break.

Reply:
MD047/single-trailing-newline: Files should end with a single newline charactermarkdownlint[MD047](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md047.md)

Having the new-line here would have been fine if we were following the above convention. We can remove the extra line @Diptipowervs